.buildkite/pipeline.yml

```diff
@@ -3,16 +3,17 @@ steps:
     command: |
       . /var/lib/buildkite-agent/.nix-profile/etc/profile.d/nix.sh
       echo "build:ci --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
+      echo "build:ci --remote_http_cache=https://storage.googleapis.com/tweag-bazel-cache-buildkite --remote_upload_local_results=true --google_default_credentials" > .bazelrc.local
```

Comment:
This duplicates the remote cache flags defined in `.bazelrc` and is likely to cause divergence and confusion.

How about the following instead:
```
echo "build --config=ci --config=ci_linux" >> .bazelrc.local
```

I did some tests with this locally, and it did enable the flags defined under `build:ci` and `build:ci_linux`.

Reply:
Actually I'm going to drop the flags from `.bazelrc` so this will be the only place where they are set. I think it's better this way because the flags are only for Buildkite builds and this way they will be "localized" and there won't be any danger that other scripts will pick them up.